FAQ.md

```diff
@@ -0,0 +1,179 @@
+
+Note: Please check the fair use (also called "fair dealing" or "copyright exception") situation in your country where it is broadcasted. This point should be important when broadcasting other contents without problems on your TV channel.
+
+Based on [#650](https://github.com/iptv-org/iptv/issues/650) and [#6654](https://github.com/iptv-org/iptv/discussions/6654) (basic information).
+
+### Is there any way to add my channel other than via this git repository?
+No, there is no such option.
+
+Based on [#2392](https://github.com/iptv-org/iptv/issues/2392).
+
+## Technical
+### Why am I asked to provide an adaptive playlist like "master.m3u8", "playlist.m3u8" or "index.m3u8"?
+An adaptive playlist is a m3u8 file that contains certain informations: the bandwidth, the quality, the type of codec used to read the channel. The player then reads the channel playlist based on your bandwidth and screen (hence the name "adaptive"). This permits you to watch your channel on perfect conditions.
+
+In the "daily update", the stream with the best supported resolution will be considered "priority" over other lower resolution streams of the same channel.
+
+In addition, we do not recommend using "chunks.m3u8" for consistency reasons.
+
+Based on [#1916](https://github.com/iptv-org/iptv/issues/1916) and [#5952](https://github.com/iptv-org/iptv/discussions/5952).
+
+### Why don't we accept Xtream-Codes servers inside our playlists?
+Xtream-Codes streams are, most of the times, pirated streams created by some people that can get satellite and/or PPV streams and that propose it to everyone for a fairer price. These types of servers are illegal and also highly unstable, since it depends either from the user who bought it or the server that hosts it.
+
+In short, they are poor quality streams created by unauthorized people.
+
+If you're unsure about if your link is from an Xtream-Codes server, you may:
+* Look at the structure of the URL. Most of them have this form : http(s)://*hostname*:25461/*username*/*password*/*channelID* (port is often 25461)
+* Remove the rest of the URL after the password mention and add either "panel_api.php?" or "player_api.php?" after the port. Replace the slashes between the username and password with "username=" and "password=". 
+
+If the link matches or answers after changing the URL, you're with an Xtream-Codes server.
+
+Based on [#5236](https://github.com/iptv-org/iptv/pull/5236), [#5401](https://github.com/iptv-org/iptv/pull/5401) and [#7576](https://github.com/iptv-org/iptv/discussions/7576) (about username and password of Xtream-Codes server).
+
+### Is it possible to add streams from Youtube, Dailymotion or Twitch?
+Yes. Due to technical limitations, it is necessary to add a [Streamlink](https://streamlink.github.io/) server link to view the content. This does not work with all streams. This does not work with all streams, so it will give an "error" status if this stream is not freely accessible.
```

Comment:
```suggestion
Yes. Due to technical limitations, it is necessary to add a [Streamlink](https://streamlink.github.io/) server link to view the content. This does not work with all streams, so it will give an "error" status if this stream is not freely accessible.
```
remove duplicate sentence

Reply:
OK. My apologies. I just had to say that geoblocked streams don't work with Streamlink.

Reply:
> OK. My apologies. I just had to say that geoblocked streams don't work with Streamlink.

It's just because you've said it in your next sentence that I have removed this one :)